### DISCUSSION TITLE
Request: Add `IO Delay %` in Node stats

### DISCUSSION THREAD

**Discussion by jeremysherriff** [dougiteixeira/proxmoxve#317] [Feature request] [answered]
I would love to be able to track the IO delay statistic in order to leverage in automations as well as track through recorder and history stats.  This would allow me to react to higher-than-expected IO delays by sending a suitable notification, curating my own startup delays based on richer node data, shutting down nice-to-have VMs, etc.

According to the API docs, the `wait` statistic of /nodes/{node}/status is the IO Delay in a % format (same scaling as the `cpu` statistic).

**Comment by jeremysherriff** [comment]
Apologies, this is a duplicate of #268 but didn't show when I searched.
I'd prefer to keep this discussion open as this one contains details of the api endpoint and attribute, as well as the scaling of the data point.

**Comment by mZ738** [comment, marked as answer]
`IO delay` is a node sensor since 5.1.0 ([release notes](https://github.com/dougiteixeira/proxmoxve/releases/tag/5.1.0)) — the share of time the CPUs spent waiting for I/O, the figure the node summary shows. On by default. Closing as done.
